src/Jackalope/ObjectManager.php

```diff
@@ -940,6 +946,10 @@ protected function executeBatch($type, $operations)
             case Operation::REMOVE_PROPERTY:
                 $this->transport->deleteProperties($operations);
                 break;
+            case Operation::SET_POLICY:
+                $this->transport->setPolicy($operations);
+                break;
+            //TODO: operation for acl changes
```

Comment:
afaik there is no different operation, the acl is always entirely overwritten. right @Alfusainey ?